samples/net/wifi/Kconfig.sysbuild

```diff
@@ -0,0 +1,11 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config WIFI_NRF70
+	default y if BOARD_NRF7002DK_NRF5340_CPUAPP || \
+		     BOARD_NRF7002DK_NRF5340_CPUAPP_NS
```

Comment:
We also need to do this for all Wi-Fi shields, else those combinations will fail :-(.

Reply:
For shields, do we have a Kconfig symbol we can depend on here?

Reply:
We have a Kconfig symbol but it's not working for sysbuild for some reason, so, I have directly updated `sample.yaml` to get twister green.